### PR TITLE
Fix/handle error routes

### LINKED ISSUE #502: Error 404 and 500
Handle some 404 and 500 errors:
- Handle error 404 when access a task not found
- Handle error 500 from newly created accounts with a notification: "There's was an error to process your request"

### Why
We have an empty screen or crash in those situations that we need to handle gracefully

### Level
Beginner

### Current behavior
We're not handling properly error 404 and 500 when access a task and when a user bank account is activated.

### Expected behavior
For 404 page, we expect a simple 404 page (a page with a header and a big 404 in the center) and a text _Not found_

### How to reproduce the user scenario

 - Create a user
 - Try to activate account (locally you will probably will get an error 500)
 - Try to access an undefined task like http://gitpay.me/#/task/any-thing

### PR DESCRIPTION
**Pull request by ShubhamCanMakeCommit**
## Description

> Successfully handles not a task route `/task/anything-here`

## Changes

- In `task.js`

## Issue

> Closes #502 

## Impacted Area

> None. Creates a new 404 component

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Type `/task/anything` in url and you should be redirected to 404 page

404-desktop

### CODE REVIEW

**Comment by ShubhamCanMakeCommit** [comment]
404-mobile
Mobile look

File: frontend/src/components/FourOFour.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        </p>
+      </div>
+      <p style={ { fontSize: 25 } }>You can go to Home page</p>
+      <div style={ {
+        fontSize: 20,
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-between'
+      } }>
+        <p><Link href='/'>Home</Link></p>
```
Comment (alexanmtz) [inline frontend/src/components/FourOFour.js:38]: @ShubhamCanMakeCommit  We need to use the internationalization here on the text strings

File: frontend/src/components/task/task.js [hunk 2]
```diff
@@ -688,7 +688,14 @@ class Task extends Component {
...
     }
-
+    // Error handling when task does not exist
+    if (task.completed && !task.values) {
+      this.props.history.push('/404')
+      return null
+    }
+    if (!task.completed) {
+      return (<div />)
+    }
```
Comment (alexanmtz) [inline frontend/src/components/task/task.js:698]: @ShubhamCanMakeCommit We should maybe handle this on actions when we try to fetch a task? And the server should return `404` and we detect the response status to actually redirect to 404 page?

File: frontend/src/main/routes.js [hunk 3]
```diff
@@ -42,6 +43,8 @@ export default props => (
         path='/task/:id/order/:order_id/status/:status'
         component={ TaskOrdersContainer }
       />
+      <Route path='/404' component={ FourOFour } />
+      <Route component={ FourOFour } />
```
Comment (alexanmtz) [inline frontend/src/main/routes.js:47]: @ShubhamCanMakeCommit why we have two routes here?

File: frontend/src/components/task/task.js (hunk 2 above)
Comment (ShubhamCanMakeCommit) [reply frontend/src/components/task/task.js:698]: But we can't call `this.props.history.push('/404')` inside [taskActions.js- syncTask()](https://github.com/worknenjoy/gitpay/blob/master/frontend/src/actions/taskActions.js#L532)
And I don't know how would I implement the other way. Maybe you can show the way on how to implement. Just a hint maybe.

File: frontend/src/main/routes.js (hunk 3 above)
Comment (ShubhamCanMakeCommit) [reply frontend/src/main/routes.js:47]: The first one is the route that is called when no task exists and the last route is the default route that would match if no other route matches. e.g. `gitpay.me/#/anything/random/here` .

File: frontend/src/components/FourOFour.js (hunk 1 above)
Comment (ShubhamCanMakeCommit) [reply frontend/src/components/FourOFour.js:38]: Yes I would implement it. I thought to show this first if you wanted any changes.

File: frontend/src/components/task/task.js (hunk 2 above)
Comment (alexanmtz) [reply frontend/src/components/task/task.js:698]: Yeah, let's keep for now this solution on the back-end and further look into a more generic solution for all 404 and 500

**Pull request merged**